README.md

```diff
@@ -1,5 +1,5 @@
 # Code for Social Good
-[![Build Status](https://travis-ci.org/Code4SocialGood/c4sg-web.svg?branch=master)](https://travis-ci.org/Code4SocialGood/c4sg-web)
+[![Build Status](https://travis-ci.org/Code4SocialGood/c4sg-web.svg?branch=master)]( https://github.com/Code4SocialGood/c4sg-services/wiki)
 
 For setup information, please see [this guide](https://docs.google.com/document/d/1CZflfQ-z52RI-3DaSf9rXxnzdIzXMfvJZzx7hlN28wg/edit). 
```

Comment:
Here is the hyperlink that should be changed, replace the google doc with wiki link